plugins/liferay-maven-plugin/src/test/java/com/liferay/maven/plugins/ServiceBuilderVerifyTest.java

```diff
@@ -26,37 +26,147 @@
+    }
+
+    
+    public void testServiceRresolveProject61() throws Exception {
+        File testDir = ResourceExtractor.simpleExtractResources(
+            getClass(), "/projects/servicebuilder/builder");
+
+        assertTrue(testDir.exists());
+
+        Verifier verifier = new Verifier(testDir.getAbsolutePath());
+
+        verifier.deleteArtifact("it", "builder", "1.0", "pom");
+        verifier.deleteArtifact("it", "builder-portlet", "1.0", "war");
+        verifier.deleteArtifact("it", "builder-portlet-service", "1.0", "jar");
+
+        verifier.setMavenDebug(true);
+
+        List<String> cliOptions = new ArrayList<String>();
+
+
+        cliOptions.add("-P");
+        cliOptions.add("v6.1.x");
+        
+        cliOptions.add( "-pl" );
+        cliOptions.add( "builder-portlet" );
+        
+        verifier.setCliOptions( cliOptions );
+
+        String[] goals = {"liferay:build-service"};
+        verifier.executeGoals( Arrays.asList( goals ) );
+        
+        verifier.verifyTextInLog(
+            "Resolved dependency project MavenProject: " +
+                "it:builder-portlet-service");       
+        verifier.resetStreams();
+    }
+    	
+	
+    public void testServiceBuilderGenerateClass62() throws Exception {
+        File testDir = ResourceExtractor.simpleExtractResources(
+            getClass(), "/projects/servicebuilder/builder");
+
+        assertTrue(testDir.exists());
 
-		Verifier verifier = new Verifier(testDir.getAbsolutePath());
+        Verifier verifier = new Verifier(testDir.getAbsolutePath());
 
-		verifier.deleteArtifact("it", "resolveProject", "1.0", "pom");
-		verifier.deleteArtifact("it", "resolveProject-portlet", "1.0", "war");
-		verifier.deleteArtifact(
-			"it", "resolveProject-portlet-service", "1.0", "jar");
-		verifier.setMavenDebug(true);
+        verifier.deleteArtifact("it", "builder", "1.0", "pom");
+        verifier.deleteArtifact("it", "builder-portlet", "1.0", "war");
+        verifier.deleteArtifact("it", "builder-portlet-service", "1.0", "jar");
 
-		List<String> cliOptions = new ArrayList<String>();
+        verifier.setMavenDebug(true);
 
-		cliOptions.add("-pl");
-		cliOptions.add("resolveProject-portlet");
+        List<String> cliOptions = new ArrayList<String>();
 
-		verifier.setCliOptions(cliOptions);
 
-		verifier.executeGoal("liferay:build-service");
+        cliOptions.add("-P");
+        cliOptions.add("v6.2.x");
+        
+        cliOptions.add( "-pl" );
+        cliOptions.add( "builder-portlet" );
+        
+        verifier.setCliOptions( cliOptions );
 
-		verifier.verifyTextInLog(
-			"Resolved dependency project MavenProject: " +
-				"it:resolveProject-portlet-service");
+        String[] goals = {"liferay:build-service"};
+        verifier.executeGoals( Arrays.asList( goals ) );
+        
+        File fooServiceUtilJavaFile = new File( verifier.getBasedir() + "\\builder-portlet-service\\src\\main\\java\\it\\service\\FooServiceUtil.java");
```

Comment:
these paths will likely not work on linux, we should use the '/' instead as those should work on both.